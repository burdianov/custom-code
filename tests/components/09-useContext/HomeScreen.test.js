import { mount } from 'enzyme';
import { UserContext } from '../../../components/09-useContext/UserContext';
import HomeScreen from './../../../components/09-useContext/HomeScreen';

describe('pruebas en <HomeScreen />', () => {
  const user = {
    name: 'John',
    email: 'john@gmail.com'
  };

  const wrapper = mount(
    <UserContext.Provider
      value={{
        user
      }}
    >
      <HomeScreen />
    </UserContext.Provider>
  );

  test('debe de mostrarse correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
