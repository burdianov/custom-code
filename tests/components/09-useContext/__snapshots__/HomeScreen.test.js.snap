// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <HomeScreen /> debe de mostrarse correctamente 1`] = `
Array [
  <h1>
    HomeScreen
  </h1>,
  <hr />,
  <pre>
    {
   "name": "John",
   "email": "john@gmail.com"
}
  </pre>,
]
`;
